frontend/components/card/discussion/CardDiscussionText.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+    <div class="flex w-full">
+      <div className="p-4">
+        <div class="flex items-center mb-2">
+          <!-- Image Logic Not Yet Implemented, For Now Placeholder is a gray circle-->
+          <div class="w-10 h-10 rounded-full bg-gray-300"></div>
+          <h1 class="font-bold responsive-h3 pl-2">{{ entry.author }}</h1>
+          <p class="text-sm pl-2">{{ entry.date.toLocaleString() }}</p>
+        </div>
+        <p>{{ entry.content }}</p>
+        <span class="ml-auto text-sm text-gray-500">Upvotes: {{ entry.votes }}</span>
+        <BtnLabeled
+            class="block"
+            :cta="true"
+            label="Votes"
+            fontSize="sm"
+            ariaLabel="components.btn-labeled.get-in-touch-aria-label"
+        />
+      </div>
+    </div>
+</template>
+  
+<script setup lang="ts">
+import type { CardDiscussionText } from "../../../types/card-discussion-text";
+  
+defineProps<{
+    isPrivate?: boolean;
+    entry: CardDiscussionText;
+}>();
+</script>
```

Comment:
Minor nit (nit pick) here that it's good to include blank lines at the end of files :) No stress though! 😎 I get my editor to add one automatically, and following the `pre-commit` setup in the [contributing PR steps](https://github.com/activist-org/activist/blob/main/CONTRIBUTING.md#pull-requests-) will make sure of this as well 😊

Reply:
I  see! I installed pre-commit, but forgot to run it on the new files. Will do so in the future, sorry about that!

Reply:
No worries, @dmsmirno! :)

Reply:
Hello, I updated the files today. Although I ran pre-commit, it seems like I still don't have a line at the end? Though I can see it on my local environment, please let me know if it's still wrong. Otherwise, I updated the button with counter and label, and included card-style and dark: background specification for the placeholder. Thanks!